appspec.yml

```
@@ -0,0 +1,16 @@
+ version: 0.0
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 1

(indentation)